db/migrate/20160907095103_add_availability_to_listing_shape.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToListingShape < ActiveRecord::Migration
+  def change
+    add_column :listing_shapes, :availability, :string, limit: 32, default: "none", after: :shipping_enabled
```

Comment:
It loses the self describing nature of the enum which affects things like migrations and adhoc debugging. I don't see a perf gain reason here to do so (we don't load listing types by enum value and even then it's just a short varchar which works ok with indexes).